docs/docusaurus.config.ts

```diff
@@ -117,10 +117,10 @@ const config: Config = {
           ],
         },
         {
-          type: 'doc',
+          type: 'docSidebar',
           position: 'left',
+          sidebarId: 'communitySidebar',
           label: 'Community',
```

Comment:
Is "community" really the right category for "telemetry"?

Reply:
Yea it's a bit strange